.github/workflows/nestml-build.yml

```diff
@@ -1,10 +1,26 @@
-name: NESTML build
+name: NESTML compatibility check with older NEST versions
+
 on: [push, pull_request]
 
+#on:
+#  schedule:
+    #        ┌───────────── minute (0 - 59)
+    #        │ ┌───────────── hour (0 - 23)
+    #        │ │ ┌───────────── day of the month (1 - 31)
+    #        │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #        │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    #        │ │ │ │ │
+    #        * * * * *
+#    - cron: '30 5 * * 0'
+
 jobs:
-  nest-3-latest:
+  build_and_test:
     runs-on: ubuntu-latest
     strategy:
+      matrix:
+        nest_branch: ["v2.20.2", "v3.3", "master"]
```

Comment:
Should we actively test for all other versions of NEST?

Reply:
Let's discuss this with @jougs after the merge. Compatibility is anyway easy to add now, but I'd like to get this in before waiting for the outcome of that discussion.